lib/active_pivot/api/project.rb

```diff
@@ -2,7 +2,7 @@ module ActivePivot
   module Api
     class Project < OpenStruct
       def self.all
-        Api::PaginatedCollection.new("/services/v5/projects.json").all.map do |remote_project|
+        Api::PaginatedCollection.new("/services/v5/projects.json?fields=:default,current_velocity,current_volatility").all.map do |remote_project|
```

Comment:
:water_buffalo: 

This chain seemed a little clunky to me, so I was messing around with ways to clean it up. Would be good to pull the endpoint out into a constant. We could also potentially allow the collection to handle some of the iterating/mapping that we're doing here, but I don't know how sold I am on it.

```
module Api
  class Project < OpenStruct
    COLLECTION_URL = "/services/v5/projects.json?fields=:default,current_velocity,current_volatility"

    def self.all
      Api::PaginatedCollection.wrap_responses(COLLECTION_URL){ |project| new(project) }
    end
  end
end

module Api
  class PaginatedCollection
    def self.wrap_responses(endpoint, params = {})
      new(endpoint, params).pages.flat_map do |page|
        yield page.parsed_response
      end
    end
  end
end
```